app/models/container_template.rb

```diff
@@ -12,4 +12,55 @@ class ContainerTemplate < ApplicationRecord
   serialize :objects, Array
 
   acts_as_miq_taggable
+
+  def instantiate(params, project = nil)
+    @ems = ext_management_system
+    @clients = {}
+    project ||= container_project.name
+
+    temp = {
+      :metadata   => {
+        :name      => name,
+        :namespace => project
+      },
+      :objects    => objects,
+      :parameters => params
+    }
+
+    processed_template = process(@ems.connect, temp)
+    create_objects(processed_template['objects'], project) if processed_template
```

Comment:
What does `create_objects` return?  The caller will need a way to determine when the operation ends so it would be better to return a handle or unique ID that can be used to query the provider.

This is what we do for other providers.  For example: [app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb#L43](https://github.com/ManageIQ/manageiq/blob/8acdc5a086f6b782d54efece8737146ab4e5d7fe/app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb#L43)